tests/test_cdat_info_generate_notebook.py

```diff
@@ -0,0 +1,50 @@
ack=3.6.1=1\n  - libffi=3.2.1=hfc679d8_5\n  - libiconv=1.15=h470a237_3\n  - libnetcdf=4.6.1=h350cafa_10\n  - libpng=1.6.35=ha92aebf_2\n  - libsodium=1.0.16=h470a237_1\n  - libssh2=1.8.0=h5b517e9_2\n  - libtiff=4.0.9=he6b73bb_2\n  - locket=0.2.0=py_2\n  - mako=1.0.7=py_1\n  - markupsafe=1.0=py36h470a237_1\n  - mccabe=0.6.1=py_1\n  - mistune=0.8.3=py36h470a237_2\n  - mkl_fft=1.0.6=py36_0\n  - mkl_random=1.0.1=py36_0\n  - mpi=1.0=mpich\n  - mpich=3.2.1=h26a2512_4\n  - msgpack-python=0.5.6=py36h2d50403_3\n  - nb_conda=2.2.1=py36_0\n  - nb_conda_kernels=2.1.1=py36_1\n  - nbconvert=5.3.1=py_1\n  - nbformat=4.4.0=py_1\n  - ncurses=6.1=hfc679d8_1\n  - netcdf-fortran=4.4.4=h71ea97b_10\n  - nettle=3.3=0\n  - nodejs=10.8.0=hfc679d8_1\n  - nose=1.3.7=py36_2\n  - notebook=5.7.0=py36_0\n  - openh264=1.7.0=0\n  - openssl=1.0.2p=h470a237_0\n  - ossuuid=1.6.2=hfc679d8_0\n  - packaging=18.0=py_0\n  - pamela=0.3.0=py36_0\n  - pandas=0.23.4=py36hf8a1672_0\n  - pandoc=2.3=0\n  - pandocfilters=1.4.2=py_1\n  - parso=0.3.1=py_0\n  - partd=0.3.8=py_1\n  - pexpect=4.6.0=py36_0\n  - pickleshare=0.7.5=py36_0\n  - pip=18.0=py36_1\n  - proj4=4.9.3=h470a237_8\n  - prometheus_client=0.3.1=py_1\n  - prompt_toolkit=1.0.15=py_1\n  - psutil=5.4.7=py36h470a237_1\n  - ptyprocess=0.6.0=py36_1000\n  - pycodestyle=2.3.1=py_1\n  - pycparser=2.19=py_0\n  - pyflakes=1.6.0=py_1\n  - pygments=2.2.0=py_1\n  - pyopenssl=18.0.0=py36_0\n  - pyparsing=2.2.2=py_0\n  - pyqt=5.6.0=py36h8210e8a_7\n  - pysocks=1.6.8=py36_2\n  - python=3.6.6=h5001a0f_0\n  - python-dateutil=2.7.3=py_0\n  - python-editor=1.0.3=py_0\n  - python-oauth2=1.1.0=py36_0\n  - pytz=2018.5=py_0\n  - pyyaml=3.13=py36h470a237_1\n  - pyzmq=17.1.2=py36hae99301_0\n  - qt=5.6.2=hd4c90f3_9\n  - qtconsole=4.4.1=py36_1\n  - readline=7.0=haf1bffa_1\n  - requests=2.19.1=py36_1\n  - send2trash=1.5.0=py_0\n  - setuptools=40.4.0=py36_1000\n  - simplegeneric=0.8.1=py_1\n  - sip=4.18.1=py36hfc679d8_0\n  - six=1.11.0=py36_1\n  - sortedcontainers=2.0.5=py_0\n  - sqlalchemy=1.2.12=py36h470a237_0\n  - sqlite=3.25.2=hb1c47c0_0\n  - tblib=1.3.2=py_1\n  - terminado=0.8.1=py36_1\n  - testpath=0.3.1=py36_1\n  - tk=8.6.8=ha92aebf_0\n  - toolz=0.9.0=py_0\n  - tornado=5.1.1=py36h470a237_0\n  - traitlets=4.3.2=py36_0\n  - udunits2=2.2.27.6=h3a4f0e9_1\n  - urllib3=1.23=py36_1\n  - wcwidth=0.1.7=py_1\n  - webencodings=0.5.1=py_1\n  - wheel=0.31.1=py36_1001\n  - widgetsnbextension=3.4.2=py36_0\n  - x264=1!152.20180717=h470a237_1\n  - xz=5.2.4=h470a237_1\n  - yaml=0.1.7=h470a237_1\n  - zeromq=4.2.5=hfc679d8_5\n  - zict=0.1.3=py_0\n  - zlib=1.2.11=h470a237_3\n  - blas=1.0=mkl\n  - intel-openmp=2019.0=118\n  - libcurl=7.61.0=hf30b1f0_0\n  - libgfortran=3.0.1=h93005f0_2\n  - mkl=2019.0=118\n  - numpy=1.15.1=py36h6a91979_0\n  - numpy-base=1.15.1=py36h8a80b8c_0\n  - pycurl=7.43.0.2=py36hdbc3d79_0\n  - pip:\n    - msgpack==0.5.6\n    - mv2==3.0.0\n    - regrid2==3.0.0\n    - sidecar==0.2.0\n    - testrunner==0.0.0\nprefix: /Users/doutriaux1/anaconda2/envs/widgets3\n\n'
```

Comment:
@doutriaux1 Is there another way to do this?  Can you put that in a file or do a `conda env export`?  Can easily break as is...